packages/terra-pills/src/subcomponents/_Pill.jsx

```diff
@@ -0,0 +1,300 @@
+/* eslint-disable jsx-a11y/role-supports-aria-props */ // lp052179
+import React, {
+  useCallback, useLayoutEffect, useRef, useState,
+} from 'react';
+import {
+  KEY_SPACE,
+  KEY_RETURN,
+  KEY_DELETE,
+  KEY_BACK_SPACE,
+} from 'keycode-js';
+import { injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+import Popup from 'terra-popup';
+import ResizeObserver from 'resize-observer-polyfill';
+import ThemeContext from 'terra-theme-context';
+import styles from './Pill.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+    * The html 'id' attribute for the pill, needed for proper keyboard interactions. (Required & must be unique)
+    */
+  id: PropTypes.string.isRequired,
+  /**
+   * The label text for the pill. (Required)
+   */
+  label: PropTypes.string.isRequired,
+  /**
+   * Secondary label text for the pill that displays a filter category
+   */
+  labelCategory: PropTypes.string,
+  /**
+   * Object returned in the `onRemove`
```

Comment:
Can you explain what do you mean by this?
```
   * Object returned in the `onRemove`
```

Reply:
changed 40ee383faaf027a3fbbd63aec56cf8e88f282029